modules/life-cycle-mandatory-upgrades.adoc

```diff
@@ -10,4 +10,5 @@ impacts the security or stability of the cluster, the customer must upgrade to t
 patch release within two link:https://access.redhat.com/articles/2623321[business days].
 
 In extreme circumstances and based on Red Hat's assessment of the CVE criticality to the
-environment, Red Hat will notify customers that they have two link:https://access.redhat.com/articles/2623321[business days] to schedule or manually update their cluster to the latest, secure patch release. In the case that an update has not been performed, Red Hat will automatically update the cluster to the latest, secure patch release to mitigate potential security breach(es) or instability. Red Hat may, at its own discretion, temporarily delay an automated update if requested by a customer through a link:https://access.redhat.com/support[support case].
\ No newline at end of file
+environment, Red Hat will notify customers that they have two link:https://access.redhat.com/articles/2623321[business days] to schedule or manually update their cluster to the latest, secure patch release.
+In the case that an update has not been performed, Red Hat will automatically update the cluster to the latest, secure patch release to mitigate potential security breach(es) or instability. Red Hat may, at its own discretion, temporarily delay an automated update if requested by a customer through a link:https://access.redhat.com/support[support case].
```

Comment:
I may be misunderstanding these two paragraphs. In the first, the user is notified that they have two business days to update. In the second paragraph if an update has not been done. it happens automatically. Can there be more of a differentiation between these two situations?

Reply:
I believe they receive two days' warning to do the upgrade themselves, after which Red Hat updates the cluster, but I'll confirm with SRE.

Reply:
Verified with SRE and updated to clarify. Thanks!